9c697d609329ca10c7067a418bba1022/update/sys_metadata_link_362f6435c3419e102f84fc84e4013121.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_metadata_link">
+    <sys_metadata_link action="INSERT_OR_UPDATE">
+        <directory>update</directory>
+        <documentkey>a00d287dc3019e102f84fc84e40131e2</documentkey>
+        <payload><![CDATA[<?xml version="1.0" encoding="UTF-8"?><record_update table="x_snc_plant_mgmt_plants">
+    <x_snc_plant_mgmt_plants action="INSERT_OR_UPDATE">
+        <date_last_watered>2024-10-05</date_last_watered>
+        <description>Renowned for digestion for human Health</description>
+        <growth_method>seed</growth_method>
+        <hardiness_zone/>
+        <image>8d7de87dc3019e102f84fc84e40131d8</image>
+        <issue/>
+        <located_outdoors>false</located_outdoors>
+        <location display_value="India">fca681f5931631003b4bb095e57ffb8d</location>
+        <name>Ajwain</name>
+        <notes/>
+        <number>PLANT0001001</number>
+        <other_issue/>
+        <patent/>
+        <plant_type display_value="Ajwain">bbad2871c3419e102f84fc84e4013197</plant_type>
+        <planted_by_date>2023-06-05</planted_by_date>
+        <status/>
+        <sys_created_by>admin</sys_created_by>
+        <sys_created_on>2024-10-05 07:29:18</sys_created_on>
+        <sys_id>a00d287dc3019e102f84fc84e40131e2</sys_id>
```

Comment:
Hi @sandeepd26 , I can see the already merged PR file in this PR(https://github.com/ServiceNowDevProgram/Plants/pull/237/files). Can you please check and remove this.